doc/Command-Reference.md

```diff
@@ -0,0 +1,3680 @@
+      * [Interface Config Commands](#interface-config-commands)
+   * [Interface Naming Mode](#interface-naming-mode)
+      * [Interface naming mode show commands](#interface-naming-mode-show-commands)
+      * [Interface naming mode config commands](#interface-naming-mode-config-commands)
+   * [IP](#ip)
+      * [IP show commands](#ip-show-commands)
+	  * [IPv6 show commands](#ipv6-show-commands)
+   * [LLDP](#lldp)
+      * [LLDP show commands](#lldp-show-commands)
+   * [Loading, Reloading And Saving Configuration](#loading-reloading-and-saving-configuration)
+      * [Load config command](#load-config-command)
+      * [Load_mgmt_config command](#load_mgmt_config-command)
+      * [Load_minigraph config command](#load_minigraph-config-command)
+      * [Reload config command](#reload-config-command)
+      * [Save config  command](#save-config--command)
+   * [Mirroring Configuration And Show](#mirroring-configuration-and-show)
+      * [Mirroring Show command](#mirroring-show-command)
+      * [Mirroring Config command](#mirroring-config-command)
+   * [NTP](#ntp)
+      * [NTP show command](#network-time-protocol-show-command)
+   * [Platform Specific Commands](#platform-specific-commands)
+   * [PortChannel Configuration And Show](#portchannel-configuration-and-show)
+      * [PortChannel Show commands](#portchannel-show-commands)
+      * [PortChannel Config commands](#portchannel-config-commands)
+   * [QoS Configuration &amp; Show](#qos-configuration--show)
+      * [QoS Show commands](#qos-show-commands)
+         * [PFC](#pfc)
+         * [Queue And Priority-Group](#queue-and-priority-group)
+      * [QoS config commands](#qos-config-commands)
+   * [Startup &amp; Running Configuration](#startup--running-configuration)
+      * [Startup Configuration command](#startup-configuration-command)
+      * [Running Configuration command](#running-configuration-command)
+   * [System State](#system-state)
+      * [Processes show commands](#processes-show-commands)
+      * [Services &amp; memory show commands](#services--memory-show-commands)
+   * [VLAN &amp; FDB](#vlan--fdb)
+      * [VLAN](#vlan)
+         * [VLAN show commands](#vlan-show-commands)
+         * [VLAN Config commands](#vlan-config-commands)
+      * [FDB](#fdb)
+         * [FDB show commands](#fdb-show-commands)
+   * [Warm Restart](#warm-restart)
+      * [Warm Restart show command](#warm-restart-show-command)
+      * [Warm Restart Config command](#warm-restart-config-command)
+   * [Watermark Configuration And Show](#watermark-configuration-and-show)
+      * [Watermark Show command](#watermark-show-command)
+      * [Watermark Config command](#watermark-config-command)
+   * [Software Installation Commands](#software-installation-commands)
+      * [SONiC Installer](#sonic-installer)
+   * [Troubleshooting Commands](#troubleshooting-commands)
+   * [Routing Stack Configuration And Show](#routing-stack-configuration-and-show)
+   
+
+# Document History

```

Comment:
Curious if we really need a "Document History" section, since this file will now be committed in the repo, and one will be able to check the history right here in GitHub. Also, the section could grow to be quite long. However, I do see how it could be useful for seeing what has changed at a quick glance.

Please share your thoughts here.

Reply:
It's just an easy way to track what is going on inside the document instead of going through the github history.